instances/latest/contentTypes/application_vnd.meshview+json.jsonld

```diff
@@ -0,0 +1,20 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/contentTypes/application_vnd.meshview+json",
+  "@type": "https://openminds.ebrains.eu/core/ContentType",
+  "dataType": null,
+  "description": "Coordinate lists for display in MeshView (https://scicrunch.org/resolver/RRID:SCR_017222)",
```

Comment:
This could be a bit more specific.
Just a suggestion, feel free to change it:

"Lists of 3-dimensional coordinate points for display in volumetric reference brain atlases in MeshView ([RRID:SCR_017222]((https://scicrunch.org/resolver/RRID:SCR_017222))."